src/Footer/ActiveCustomer/ActiveCustomer.jsx

```diff
@@ -1,14 +1,14 @@
-
-
-function ActiveCustomer() {
+import "./ActiveCustomer.css";
+export const ActiveCustomer = ({ arrayToRender }) => {
```

Comment:
Keep an empty line between the component and imports

Reply:
ok